templates/cloud_init/cloud_init.client.yml

```diff
@@ -0,0 +1,367 @@
+#cloud-config
+
+final_message: "<> DONE: Cloud-Init COMPLETE. The VM is UP and READY ($UPTIME seconds elapsed)."
+
+write_files:
+  - encoding: b64
+    path: /root/.ssh/id_rsa
+    permissions: '0600'
+    content: '<id_rsa>'
+
+  - path: /opt/setup_environment.sh
+    permissions: '0755'
+    content: |
+      #!/bin/bash
+
+      function error() {
+        >&2 echo "echo >< ERROR: $* See log for details."
+        exit 1
+      }
+
+      export AZURE_SUBSCRIPTION='<sub_id>'
+      export ARM_SUBSCRIPTION_ID='<sub_id>'
+      export ARM_CLIENT_ID='<arm_client_id>'
+      export ARM_CLIENT_SECRET='<arm_client_scrt>'
+      export ARM_TENANT_ID='<arm_tenant_id>'
+      export REGION='<region>'
+      export RESOURCE_GROUP_NAME='<resource_group_name>'
+      export CONTROLLER_IMAGE='<controller_image>'
+      export VFXT_IMAGE='<vfxt_image>'
+      export AVERE_ADMIN_PW='<avere_admin_pw>'
+      export VNET_NAME='<vnet_name>'
+      export RUN_TERRAFORM_FILE='<run_terraform_file>'
+      export LINUX_USER='<vm_admin_user>'
```

Comment:
`LINUX_USER` is not used in this shell script.

_General comment to review variables and only set/require what's used in all shell scripts in this cloud-init file._

Reply:
removed